### PR TITLE
[zh-cn]sync strict-cost-enforcement-*

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/strict-cost-enforcement-for-vap.md
content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/strict-cost-enforcement-for-webhooks.md

### CODE REVIEW

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/strict-cost-enforcement-for-vap.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+stages:
+  - stage: beta
+    defaultValue: false
+    fromVersion: "1.31"
+---
+  
+<!--
+Apply strict CEL cost validation for ValidatingAdmissionPolicies.
+-->
+应用严格的 CEL 成本验证来验证录取政策。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/strict-cost-enforcement-for-vap.md:18]:
```suggestion
对 ValidatingAdmissionPolicy 应用严格的 CEL 成本验证。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/strict-cost-enforcement-for-webhooks.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  - stage: beta
+    defaultValue: false
+    fromVersion: "1.31"
+---
+  
+<!--
+Apply strict CEL cost validation for `matchConditions` within
+admission webhooks.
+-->
+对 ValidatingAdmissionPolicies 应用严格的 CEL 成本验证。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/strict-cost-enforcement-for-webhooks.md:19]:
```suggestion
在准入 Webhook 内对 `matchConditions` 应用严格的 CEL 成本验证。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/strict-cost-enforcement-for-vap.md (hunk 1 above)
Comment (tengqm) [reply content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/strict-cost-enforcement-for-vap.md:18]: cost 在这个语境指的是 performance cost，翻译为“开销”合适一些。

**Review by windsonsea — COMMENTED** [review]
/approve

**Pull request merged**